pages/patterns/create-a-profile/social-security-number.md

```diff
@@ -116,12 +116,12 @@ Provide a single text field to enter the Social Security Number.
 ## References
 - 8 recommendations for creating effective input fields. (n.d.) Retrieved on July 20, 2022, from [https://baymard.com/learn/input-fields](https://baymard.com/learn/input-fields)
 - Consider using localized input masks for ‘phone’ and other restricted inputs. (November 28, 2017) Retrieved on July 20, 2022, from [https://baymard.com/blog/input-masking-form-field](https://baymard.com/blog/input-masking-form-field)
-- Masked field. (n.d.) Retrieved on July 20, 2022, from [https://design.cms.gov/components/masked-field](https://design.cms.gov/components/masked-field)
+- Masked field. (n.d.) Retrieved on July 20, 2022, from [https://design.cms.gov/v/5.0.2/components/masked-field](https://design.cms.gov/v/5.0.2/components/masked-field)
```

Comment:
Could not find the masked field component in the current version, but found it in 5.0.2 for core. Proposing we point to that instead, but I am not sure if this is an accurate reference.

Reply:
I found a masked field in the latest version nested under their text field component!

https://design.cms.gov/components/text-field/masked-field/?theme=core

Reply:
Perfect! Thanks, @mahoneycm

Updated in [93a4bf8](https://github.com/uswds/uswds-site/pull/2128/commits/93a4bf845d992e0d3344d9467b2e32b9231e8d53)